CMakeLists.txt

```diff
@@ -162,13 +175,8 @@ if(NOT CURL_LIBRARY AND EXISTS "$ENV{CURL_INSTALL_PREFIX}/lib/libcurl.a")
   SET(CMAKE_USE_SYSTEM_CURL TRUE)
   SET(CURL_NO_CURL_CMAKE ON)
 endif()
-if (NOT crypto_LIBRARY AND CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND EXISTS "$ENV{OPENSSL_INSTALL_PREFIX}/lib64/libcrypto.a")
-  SET(crypto_LIBRARY "$ENV{OPENSSL_INSTALL_PREFIX}/lib64/libcrypto.a" CACHE INTERNAL "")
-elseif(NOT crypto_LIBRARY AND NOT CMAKE_SYSTEM_PROCESSOR STREQUAL "x86_64" AND EXISTS "$ENV{OPENSSL_INSTALL_PREFIX}/lib/libcrypto.a")
-  SET(crypto_LIBRARY "$ENV{OPENSSL_INSTALL_PREFIX}/lib/libcrypto.a" CACHE INTERNAL "")
-endif()
-if (NOT crypto_INCLUDE_DIR AND EXISTS "$ENV{OPENSSL_INSTALL_PREFIX}/include")
-  SET(crypto_INCLUDE_DIR "$ENV{OPENSSL_INSTALL_PREFIX}/include" CACHE INTERNAL "")
+if(NOT AWSSDK_ROOT)
+  SET(AWSSDK_ROOT "$ENV{AWS_INSTALL_PREFIX}")
```

Comment:
This approach seems disconnected from the real aim here. CMake should not be using magic environment settings that are set by the CI build scripts.

Reply:
AWSSDK_ROOT needs to be defined for CMake to find the aws sdk. Setting environment variables for paths matches what we do for other components (e.g. cutensornet and custatevec libraries) and ensures that we can build in different environments. Keep in mind that cmake is not necessarily directly invoked, so passing this to cmake is not a great option. Additionally, CMake itself will actually pick this up from environment variables (if you define AWSSDK_ROOT as environment variable, CMake should pick that up as well).

Reply:
I don't really agree.

cmake is invoked directly. I do it multiple times a day, in fact. I do not read through shell scripts to try and figure out special environment variables and would not expect anyone else to do so.

cmake should use find to determine if the aws sdk is installed and there should be some standardish cmake variable settings to override where find looks. e.g., `-DAWS_SDK_DIR=/path`.

Reply:
Discussed offline to change all *INSTALL_PREFIX variables to the *ROOT variable in a separate PR. 
Keeping the default configuration to build the AWS integration, since we don't have a full CI setup to ensure these pieces are included if the build too willingly just skips them. Any AWS components can be omitted by setting CUDAQ_ENABLE_BRAKET_BACKENDS to false. In that case, the everything works without the need to install the aws sdk beforehand.